library/perl

```diff
@@ -1,101 +1,133 @@
 Maintainers: Peter Martini <PeterCMartini@GMail.com> (@PeterMartini),
              Zak B. Elep <zakame@cpan.org> (@zakame)
 GitRepo: https://github.com/perl/docker-perl.git
-GitCommit: f70e8ace49994efef8e90cbf730554a3e3201da7
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
+GitCommit: a0b67543fd56367c403a2f063dfed1a69fdda390
+Architectures: amd64, arm32v7, arm64v8, i386
 
 Tags: 5.36.0, 5.36, 5, latest, 5.36.0-bullseye, 5.36-bullseye, 5-bullseye, bullseye
```

Comment:
Sorry, sometimes I forget that I'm working from a huge amount of assumed context that not everyone has :joy:

For background, the reason this PR made me think about adding a `stable` alias is not just that I looked up a lot of details of "what does Perl label releases as" but also because adding a `stable` alias is an extra clue to users looking at the list of tags that 5.37 might _not_ be what they want to choose, and that 5.36 is probably more what they desire (assuming they don't know exactly which version of Perl they want/need _and_ thus also likely don't know the Perl convention around odd-numbered minor releases being unstable/development "preview" releases).

Reply:
Yeah, adding `stable` is also easy enough with our current automation, thanks for suggesting! ⚙️

Reply:
We should now have `stable` tags in https://github.com/docker-library/official-images/pull/13928/files#diff-40ed9e38417ffb2e97e136f9de487c6cdb45a2c5c1f8667b871cb8573b12f4a1L7-R36